resources/scripts/admin/stores/customer.js

```diff
@@ -174,6 +174,32 @@ export const useCustomerStore = (useWindow = false) => {
         })
       },
 
+      importCustomers(data) {
+        return new Promise((resolve, reject) => {
+          axios
+            .post('/api/v1/customers/import', data, {
+              headers: {
+                'Content-Type': 'multipart/form-data',
+              }
+            })
+            .then((response) => {
+              this.fetchCustomers({})
```

Comment:
This does not work. I want to reload the table with all customers once the import is done. How would I do that?